projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -4735,6 +4735,142 @@ public string FatherIndustry
             set => SetIndustry("FTH", value);
         }
 
+        /// <summary>Mother Height.</summary>
+        /// <value>the height of the mother, given in inches</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.MotherHeight = 55;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Mother's Height: {ExampleBirthRecord.MotherHeight}");</para>
+        /// </example>
+        [Property("MotherHeight", Property.Types.Int32, "Mother Prenatal", "Mother's Height.", false, BFDR.IGURL.ObservationMotherHeight, true, 134)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='8302-2')", "")]
+        public int? MotherHeight
+        {
+            get
+            {
+              var entry = Bundle.Entry.Where(e => e.Resource is Observation obs && CodeableConceptToDict(obs.Code)["code"] == "8302-2").FirstOrDefault();
+              if (entry != null)
+              {
+                  Observation observation = (Observation)entry.Resource;
+                  return (int?)(observation?.Value as Hl7.Fhir.Model.Quantity)?.Value;
+              }
+            return null;
+            }
+
+            set
+            {
+              Observation obs = GetOrCreateObservation("8302-2", CodeSystems.LOINC, BFDR.ProfileURL.ObservationMotherHeight, MOTHER_PRENATAL_SECTION, Mother.Id);
+              obs.Category.Add(new CodeableConcept(CodeSystems.ObservationCategory, "vital-signs"));
+              string unit = "in_i";
+              // Create an empty quantity if needed
+              if (obs.Value == null || obs.Value as Quantity == null)
+              {
+                obs.Value = new Hl7.Fhir.Model.Quantity();
+              }
+              // Set the properties of the value individually to preserve any existing obs.Value.Extension entries
+              if (value != null)
+              {
+                (obs.Value as Quantity).Value = (int)value;
+                (obs.Value as Quantity).Unit = unit;
+                (obs.Value as Quantity).Code = unit;
+              }
+            }
+        }
+
+        private string GetHeightEditFlagHelper()
```

Comment:
this doesn't need to be in a helper function since it's only called in one place